index.html

```diff
@@ -85,12 +85,12 @@ <h1 class="greet">
         </div>
         <div class="project-container" id="project_2">
           <div class="work-img">
-            <img src="assets/Snapshoot Portfoliowrok2.png">
+            <img src="assets/Snapshoot Portfoliowrok2.png" alt="work-card">
           </div>
           <div class="project-title"></div>
           <div class="project-title">Tonic</div>
           <div class="some-list">
-            <img src="assets/Frame 2.png">
+            <img src="assets/Frame 2.png" alt="">
```

Comment:
- [ ] Kindly add an alt to this attribute for other devices to be able to access it.